config/packages/prooph_event_store.yml

```diff
@@ -1,22 +1,31 @@
 prooph_event_store:
     stores:
         user_store:
-            event_store: Prooph\EventStore\Pdo\MySqlEventStore
+            event_store: 'app.event_store.default'
             repositories:
-                user_collection:
-                    repository_class: App\Infrastructure\User\Repository\EventStoreUserCollection
+                App\Infrastructure\User\Repository\EventStoreUserCollection:
```

Comment:
This is important, this key will be used to name the repository service. Using something else than a FQCN will prevent autowiring to work.

In your case, a `user_collection` service was defined, thus making it impossible to be "auto injected" into the SignupHandler.

This is kind of strange to allow non FQCN key, this should be discussed with the maintainers IMO. Thanks for pointing out this fact.